python/eups/Eups.py

```diff
@@ -2546,8 +2546,8 @@ def cleanup(*args):
                     if not os.path.exists(pathOut):
                         differences += ["Adding %s" % pathOut]
                     else:
-                        crcOld = zlib.crc32("".join(open(fileNameIn).readlines()))
-                        crcNew = zlib.crc32("".join(open(pathOut).readlines()))
+                        crcOld = zlib.crc32(b"".join(open(fileNameIn, "rb").readlines()))
```

Comment:
I think it might be clearer to do:
```python
crcOld = zlib.crc32(open(fileNameIn, "rb").read()
```
as that is more clearly a bytes operation as opposed to readlines returning a list and having to join with a blank byte object.